scripts/import.ts

```diff
@@ -252,6 +432,7 @@ const printUsage = () => {
     
     Commands:
       - parse-tokens
```

Comment:
`parse-tokens` command was deleted in L288, so should be changed to `import-colors-semantic`?

Reply:
Good catch! Fixed it.